crates/astria-celestia-jsonrpc-client/k8s/deployment.yml

```diff
@@ -0,0 +1,108 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: astria-celestia-jsonrpc-client-test
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: astria-celestia-jsonrpc-client-test
+  labels:
+    app.kubernetes.io/name: astria-celestia-jsonrpc-client-test
+  namespace: astria-celestia-jsonrpc-client-test
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: astria-celestia-jsonrpc-client-test
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: astria-celestia-jsonrpc-client-test
+    spec:
+      securityContext:
+        runAsUser: 10001
+        fsGroup: 10001
+        fsGroupChangePolicy: "OnRootMismatch"
+      initContainers:
+        - name: init-celestia-app
+          command: 
+          - /scripts/init-celestia-appd.sh
+          image: "ghcr.io/celestiaorg/celestia-app:v1.0.0-rc5"
+          volumeMounts:
+            - mountPath: /scripts
+              name: celestia-appd-scripts-volume
+              readOnly: true
+            - mountPath: /home/celestia
+              name: celestia-home
+          envFrom:
+            - configMapRef:
+                name: celestia-config
+        - command:
+          - /scripts/init-bridge.sh
+          name: init-bridge
+          # image: "ghcr.io/astriaorg/celestia-node:test-sha-07fa3e7"
+          image: "celestia-node:v0.11.0-rc5-test"
```

Comment:
Cool. I have started this the [test-images repo](https://github.com/astriaorg/test-images/tree/celestia-node) (celestia-node branch) because forking their repos to create forked container images is unsustainable.

So this will just be a collection of slightly modified container files.